tests/api_tests/users/test_views.py

```diff
@@ -203,6 +204,313 @@ def test_get_projects_logged_in_as_different_user(self):
         assert_not_in(self.deleted_project_user_one._id, ids)
 
 
+class TestUserUpdate(ApiTestCase):
+
+    def setUp(self):
+        super(TestUserUpdate, self).setUp()
+        self.user_one = UserFactory.build()
+        self.user_one.set_password('justapoorboy')
+        self.user_one.fullname = 'Martin Luther King Jr.'
+        self.user_one.given_name = 'Martin'
+        self.user_one.family_name = 'King'
+        self.user_one.suffix = 'Jr.'
+        self.user_one.social['github'] = 'userOnegitHub'
+        self.user_one.social['scholar'] = 'userOneScholar'
+        self.user_one.social['personal'] = 'http://www.useronepersonalwebsite.com'
+        self.user_one.social['twitter'] = 'userOneTwitter'
+        self.user_one.social['linkedIn'] = 'userOneLinkedIn'
+        self.user_one.social['impactStory'] = 'userOneImpactStory'
+        self.user_one.social['orcid'] = 'userOneOrcid'
+        self.user_one.social['researcherId'] = 'userOneResearcherId'
+
+        self.user_one.jobs = [
+            {
+                'startYear': '1995',
+                'title': '',
+                'startMonth': 1,
+                'endMonth': None,
+                'endYear': None,
+                'ongoing': False,
+                'department': '',
+                'institution': 'Waffle House'
+            }
+        ]
+        self.user_one.save()
+        self.auth_one = (self.user_one.username, 'justapoorboy')
+        self.user_one_url = "/v2/users/{}/".format(self.user_one._id)
+
+        self.user_two = UserFactory.build()
+        self.user_two.set_password('justapoorboy')
+        self.user_two.save()
+        self.auth_two = (self.user_two.username, 'justapoorboy')
+
+        self.new_fullname = 'el-Hajj Malik el-Shabazz'
+        self.new_given_name = 'Malcolm'
+        self.new_family_name = 'X'
+        self.new_suffix = 'Sr.'
+        self.blank_suffix = ''
+        self.new_gitHub = 'newgitHub'
```

Comment:
These instance variables are also fine, but unnecessary because you can access them all from the `new_user_data` dictionary.

```python
new_twitter = self.new_user_data['twitter']
```